analytics/management/commands/populate_analytics_db.py

```diff
@@ -107,17 +106,15 @@ def insert_fixture_data(stat, fixture_data, table):
         FillState.objects.create(property=stat.property, end_time=last_end_time,
                                  state=FillState.DONE)
 
-        # We use client names that start with _ to make it easy to
-        # drop just these data manually without affecting other data.
-        website = Client.objects.create(name='_Website')
-        old_desktop = Client.objects.create(name='_Old desktop app')
-        android = Client.objects.create(name='_Android app')
-        iOS = Client.objects.create(name='_Old iOS app')
-        react_native = Client.objects.create(name='_New iOS app')
-        API = Client.objects.create(name='_Python API')
-        barnowl = Client.objects.create(name='_Barnowl')
-        unused = Client.objects.create(name='_Unused webhook')
-        long_webhook = Client.objects.create(name='_Webhook with loooooooong name')
+        website, created = Client.objects.get_or_create(name='Website')
```

Comment:
Wasn't intentional, I'll change it to match stuff in `analytics/views.client_label_map`.